components/sst_importer/src/sst_importer.rs

```diff
@@ -15,16 +15,17 @@ use collections::HashSet;
 use dashmap::{mapref::entry::Entry, DashMap};
 use encryption::{DataKeyManager, FileEncryptionInfo, MultiMasterKeyBackend};
 use encryption_export::create_async_backend;
+use engine_rocks::{get_env, RocksSstReader};
 use engine_traits::{
     name_to_cf, util::check_key_in_range, CfName, IterOptions, Iterator, KvEngine, RefIterable,
     SstCompressionType, SstExt, SstMetaInfo, SstReader, SstWriter, SstWriterBuilder, CF_DEFAULT,
     CF_WRITE,
 };
 use external_storage::{
     compression_reader_dispatcher, encrypt_wrap_reader, wrap_with_checksum_reader_if_needed,
-    ExternalStorage, RestoreConfig,
+    ExternalStorage, RestoreConfig, RestoreConfig as ExternalRestoreConfig,
```

Comment:
merge `RestoreConfig, RestoreConfig as ExternalRestoreConfig,` to `RestoreConfig`